api/src/Filter/MaterialItemPeriodFilter.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace App\Filter;
```

Comment:
Den einen anderen custom Filter den wir schon haben habe ich in `App\Doctrine\Filter` abgelegt, weil das API Platform auch so macht. Wo denkst du ist es sinnvoller?